src/Components/Publishing/Header/FeatureHeader.tsx

```diff
@@ -23,6 +23,11 @@ interface FeatureHeaderProps {
   vertical?: any
 }
 
+interface DivProps {
+  height?: string
+  src?: string
+}
```

Comment:
This isn’t really correct. It’s now being applied to all those styled divs, but they really only refer to 2 instances and e.g. `FeatureHeaderContainer` really needs the `height` prop, it’s not optional. Is there no way to specify the specific typing when you `extend` the div?